router/nginx.conf

```diff
@@ -125,4 +125,40 @@ http {
       return 200 'OK';
     }
   }
+
+  # Wellcome Images deprecation
+  map $arg_MIRO $redirect_uri {
+    ""      https://wellcomecollection.org/works?wellcomeImagesUrl=$request_uri;
+    default https://iiif.wellcomecollection.org/image/$arg_MIRO.jpg/full/125,/0/default.jpg;
+  }
+
+  server {
+    listen      80;
+    server_name router.wellcomeimages.org; # TODO: update to wellcomeimages.org when ready to split traffic
+
+    location /indexplus/lightbox.html { # Don't break lightboxes for as long as possible
+      proxy_pass       https://prev.wellcomeimages.org;
+      proxy_set_header Host $host;
+      proxy_set_header HTTP_X_FORWARDED_PROTO https;
+    }
+
+    location / {
+      rewrite '^/indexplus/page/About Us\.html$' https://wellcomecollection.org/what-we-do/about-wellcome-collection?wellcomeImagesUrl=$request_uri permanent;
```

Comment:
I wonder if there is a more generic QS parameter, `redirectedFrom`?
Just thinking if we need it again for analytics.